community/detectors/goanywhere_cve_2023_0669/src/main/java/com/google/tsunami/plugins/detectors/goanywhere/Cve20230669VulnDetector.java

```diff
@@ -0,0 +1,141 @@
+    try {
+      HttpResponse httpResponse =
+          httpClient.send(
+              post(licenseUrl)
+                  .setHeaders(
+                      HttpHeaders.builder()
+                          .addHeader(
+                              "Content-Type",
+                              "application/x-www-form-urlencoded; " + "charset=UTF-8")
+                          .addHeader(COMMAND_HEADER, commandToInject)
+                          .build())
+                  .setRequestBody(ByteString.copyFromUtf8(deserialized))
+                  .build(),
+              networkService);
```

Comment:
```suggestion
              networkService);
      Uninterruptibles.sleepUninterruptibly(Duration.ofSeconds(3));
```
Add some sleep time here just in case